### ISSUE TITLE
Add possibility to chose crosshairMarker color, so it will be independent from line-series color

### ISSUE THREAD

**Issue by GinetSan** [tradingview/lightweight-charts#310]
Solution that would be nice its additional field "crosshairMarkerColor" to addLineSeries settings
```
const lineSeries = chart.addLineSeries({
    crosshairMarkerVisible: true,
    crosshairMarkerRadius: 6,
    crosshairMarkerColor: '#111',
});
```

**Comment by mecm1993** [comment]
I would like to give it a try

**Comment by timocov** [comment]
The fix is just published in [3.3.0 version](https://github.com/tradingview/lightweight-charts/releases/tag/3.3.0).
